projects/plugins/jetpack-boost/docker/boost-dev/.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_ROOT_PASSWORD=jetpackboostrocks
```

Comment:
If you can manage to work with Jetpack's docker environment instead of making your own, that would probably be better.

I think Boost will already be made available in Jetpack's docker environment just for the fact of `projects/plugins/jetpack-boost/` existing.